Mapper-Unit Tests random fails

The BuildMapperTest.mapBuildToBuildResponse:54 Unit-Test seems to randomly fail during some builds. See the below job log for one failing build as an example.
Please check this as it's annoying to have builds fail unrelated to the actual PR.

```
:: Spring Boot ::        (v2.1.1.RELEASE)
2019-01-16 09:02:58.339  INFO 6439 --- [           main] d.f.i.m.MaypadBackendApplicationTests    : Starting MaypadBackendApplicationTests on travis-job-46141124-7e2c-4c3a-8c57-a7e6e65c2024 with PID 6439 (started by travis in /home/travis/build/juliantodt/maypad/backend)
2019-01-16 09:02:58.347  INFO 6439 --- [           main] d.f.i.m.MaypadBackendApplicationTests    : No active profile set, falling back to default profiles: default
2019-01-16 09:02:59.333  INFO 6439 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data repositories in DEFAULT mode.
2019-01-16 09:02:59.535  INFO 6439 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 190ms. Found 2 repository interfaces.
2019-01-16 09:03:00.404  INFO 6439 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2019-01-16 09:03:00.644  INFO 6439 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2019-01-16 09:03:00.787  INFO 6439 --- [           main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [
	name: default
	...]
2019-01-16 09:03:00.895  INFO 6439 --- [           main] org.hibernate.Version                    : HHH000412: Hibernate Core {5.3.7.Final}
2019-01-16 09:03:00.903  INFO 6439 --- [           main] org.hibernate.cfg.Environment            : HHH000206: hibernate.properties not found
2019-01-16 09:03:01.179  INFO 6439 --- [           main] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.0.4.Final}
2019-01-16 09:03:01.520  INFO 6439 --- [           main] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.H2Dialect
2019-01-16 09:03:03.221  INFO 6439 --- [           main] o.h.t.schema.internal.SchemaCreatorImpl  : HHH000476: Executing import script 'org.hibernate.tool.schema.internal.exec.ScriptSourceInputNonExistentImpl@1fc5c0b2'
2019-01-16 09:03:03.237  INFO 6439 --- [           main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2019-01-16 09:03:04.076  INFO 6439 --- [           main] d.f.i.m.MaypadBackendApplicationTests    : Started MaypadBackendApplicationTests in 6.186 seconds (JVM running for 7.952)
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 7.456 s - in de.fraunhofer.iosb.maypadbackend.MaypadBackendApplicationTests
[INFO] Running de.fraunhofer.iosb.maypadbackend.dtos.mapper.DeploymentMapperTest
2019-01-16 09:03:04.221  INFO 6439 --- [           main] .b.t.c.SpringBootTestContextBootstrapper : Neither @ContextConfiguration nor @ContextHierarchy found for test class [de.fraunhofer.iosb.maypadbackend.dtos.mapper.DeploymentMapperTest], using SpringBootContextLoader
2019-01-16 09:03:04.222  INFO 6439 --- [           main] o.s.t.c.support.AbstractContextLoader    : Could not detect default resource locations for test class [de.fraunhofer.iosb.maypadbackend.dtos.mapper.DeploymentMapperTest]: no resource found for suffixes {-context.xml, Context.groovy}.
2019-01-16 09:03:04.222  INFO 6439 --- [           main] t.c.s.AnnotationConfigContextLoaderUtils : Could not detect default configuration classes for test class [de.fraunhofer.iosb.maypadbackend.dtos.mapper.DeploymentMapperTest]: DeploymentMapperTest does not declare any static, non-private, non-final, nested classes annotated with @Configuration.
2019-01-16 09:03:04.269  INFO 6439 --- [           main] .b.t.c.SpringBootTestContextBootstrapper : Found @SpringBootConfiguration de.fraunhofer.iosb.maypadbackend.MaypadBackendApplication for test class de.fraunhofer.iosb.maypadbackend.dtos.mapper.DeploymentMapperTest
2019-01-16 09:03:04.278  INFO 6439 --- [           main] .b.t.c.SpringBootTestContextBootstrapper : Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener]
2019-01-16 09:03:04.281  INFO 6439 --- [           main] .b.t.c.SpringBootTestContextBootstrapper : Using TestExecutionListeners: [org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@70819ba8, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@446a692f, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@283ecb4b, org.springframework.test.context.support.DirtiesContextTestExecutionListener@a30dbc0, org.springframework.test.context.transaction.TransactionalTestExecutionListener@76104df5, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@61246109, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@3f9e8af5, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@337cb81d, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@47772462, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@59929ac, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@4e9bd2c8]
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.289 s - in de.fraunhofer.iosb.maypadbackend.dtos.mapper.DeploymentMapperTest
[INFO] Running de.fraunhofer.iosb.maypadbackend.dtos.mapper.BuildMapperTest
2019-01-16 09:03:04.560  INFO 6439 --- [           main] .b.t.c.SpringBootTestContextBootstrapper : Neither @ContextConfiguration nor @ContextHierarchy found for test class [de.fraunhofer.iosb.maypadbackend.dtos.mapper.BuildMapperTest], using SpringBootContextLoader
2019-01-16 09:03:04.561  INFO 6439 --- [           main] o.s.t.c.support.AbstractContextLoader    : Could not detect default resource locations for test class [de.fraunhofer.iosb.maypadbackend.dtos.mapper.BuildMapperTest]: no resource found for suffixes {-context.xml, Context.groovy}.
2019-01-16 09:03:04.562  INFO 6439 --- [           main] t.c.s.AnnotationConfigContextLoaderUtils : Could not detect default configuration classes for test class [de.fraunhofer.iosb.maypadbackend.dtos.mapper.BuildMapperTest]: BuildMapperTest does not declare any static, non-private, non-final, nested classes annotated with @Configuration.
2019-01-16 09:03:04.569  INFO 6439 --- [           main] .b.t.c.SpringBootTestContextBootstrapper : Found @SpringBootConfiguration de.fraunhofer.iosb.maypadbackend.MaypadBackendApplication for test class de.fraunhofer.iosb.maypadbackend.dtos.mapper.BuildMapperTest
2019-01-16 09:03:04.580  INFO 6439 --- [           main] .b.t.c.SpringBootTestContextBootstrapper : Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener]
2019-01-16 09:03:04.583  INFO 6439 --- [           main] .b.t.c.SpringBootTestContextBootstrapper : Using TestExecutionListeners: [org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@1f6d7e7c, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@20518250, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@7836c79, org.springframework.test.context.support.DirtiesContextTestExecutionListener@5ccc971e, org.springframework.test.context.transaction.TransactionalTestExecutionListener@28269c65, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@4c9d833, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@18b58c77, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@20f99c18, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@3c20e9d6, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@76e2a621, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@42805abe]
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.159 s <<< FAILURE! - in de.fraunhofer.iosb.maypadbackend.dtos.mapper.BuildMapperTest
[ERROR] mapBuildToBuildResponse(de.fraunhofer.iosb.maypadbackend.dtos.mapper.BuildMapperTest)  Time elapsed: 0.001 s  <<< FAILURE!
org.junit.ComparisonFailure: expected:<...19-01-16T09:03:04.58[6]> but was:<...19-01-16T09:03:04.58[5]>
	at de.fraunhofer.iosb.maypadbackend.dtos.mapper.BuildMapperTest.mapBuildToBuildResponse(BuildMapperTest.java:54)
[INFO] Running de.fraunhofer.iosb.maypadbackend.dtos.mapper.CommitMapperTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.005 s - in de.fraunhofer.iosb.maypadbackend.dtos.mapper.CommitMapperTest
[INFO] Running de.fraunhofer.iosb.maypadbackend.dtos.mapper.BranchMapperTest
2019-01-16 09:03:04.629  INFO 6439 --- [           main] .b.t.c.SpringBootTestContextBootstrapper : Neither @ContextConfiguration nor @ContextHierarchy found for test class [de.fraunhofer.iosb.maypadbackend.dtos.mapper.BranchMapperTest], using SpringBootContextLoader
2019-01-16 09:03:04.636  INFO 6439 --- [           main] o.s.t.c.support.AbstractContextLoader    : Could not detect default resource locations for test class [de.fraunhofer.iosb.maypadbackend.dtos.mapper.BranchMapperTest]: no resource found for suffixes {-context.xml, Context.groovy}.
2019-01-16 09:03:04.636  INFO 6439 --- [           main] t.c.s.AnnotationConfigContextLoaderUtils : Could not detect default configuration classes for test class [de.fraunhofer.iosb.maypadbackend.dtos.mapper.BranchMapperTest]: BranchMapperTest does not declare any static, non-private, non-final, nested classes annotated with @Configuration.
2019-01-16 09:03:04.638  INFO 6439 --- [           main] .b.t.c.SpringBootTestContextBootstrapper : Found @SpringBootConfiguration de.fraunhofer.iosb.maypadbackend.MaypadBackendApplication for test class de.fraunhofer.iosb.maypadbackend.dtos.mapper.BranchMapperTest
2019-01-16 09:03:04.642  INFO 6439 --- [           main] .b.t.c.SpringBootTestContextBootstrapper : Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener]
2019-01-16 09:03:04.643  INFO 6439 --- [           main] .b.t.c.SpringBootTestContextBootstrapper : Using TestExecutionListeners: [org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@4e26040f, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@1782896, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@8bde368, org.springframework.test.context.support.DirtiesContextTestExecutionListener@5065bdac, org.springframework.test.context.transaction.TransactionalTestExecutionListener@2e9dcdd3, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@6e617c0e, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@9a07409, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@7455204c, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@34e68840, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@510203de, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@5b0835cb]
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.06 s - in de.fraunhofer.iosb.maypadbackend.dtos.mapper.BranchMapperTest
[INFO] Running de.fraunhofer.iosb.maypadbackend.dtos.mapper.ProjectMapperTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.001 s - in de.fraunhofer.iosb.maypadbackend.dtos.mapper.ProjectMapperTest
[INFO] Running de.fraunhofer.iosb.maypadbackend.dtos.mapper.ProjectgroupMapperTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.001 s - in de.fraunhofer.iosb.maypadbackend.dtos.mapper.ProjectgroupMapperTest
2019-01-16 09:03:04.887  INFO 6439 --- [       Thread-2] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2019-01-16 09:03:04.889  INFO 6439 --- [       Thread-2] .SchemaDropperImpl$DelayedDropActionImpl : HHH000477: Starting delayed evictData of schema as part of SessionFactory shut-down'
2019-01-16 09:03:04.935  INFO 6439 --- [       Thread-2] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2019-01-16 09:03:04.947  INFO 6439 --- [       Thread-2] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   BuildMapperTest.mapBuildToBuildResponse:54 expected:<...19-01-16T09:03:04.58[6]> but was:<...19-01-16T09:03:04.58[5]>
[INFO] 
[ERROR] Tests run: 7, Failures: 1, Errors: 0, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 15.338 s
[INFO] Finished at: 2019-01-16T09:03:05Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.1:test (default-test) on project maypad-backend: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/travis/build/juliantodt/maypad/backend/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

--- a/backend/src/test/java/de/fraunhofer/iosb/maypadbackend/dtos/mapper/BuildMapperTest.java
+++ b/backend/src/test/java/de/fraunhofer/iosb/maypadbackend/dtos/mapper/BuildMapperTest.java
@@ -51,7 +51,7 @@ public class BuildMapperTest {
         assertThat(response).isNotNull();
         assertThat(response.getStatus()).isEqualTo(buildStatus);
         assertThat(response.getTimestamp()).isEqualTo(buildDate);
-        assertThat(response.getTimestamp()).isEqualTo(commitTimestamp);
+        assertThat(response.getCommit().getTimestamp()).isEqualTo(commitTimestamp);
         assertThat(response.getCommit().getCommitMessage()).isEqualTo(commitMessage);
         assertThat(response.getCommit().getCommitIdentifier()).isEqualTo(commitIdentifier);
         assertThat(response.getCommit().getAuthor()).isEqualTo(

